core/templates.py

```diff
@@ -0,0 +1,20 @@
+from flask import request, Markup
+from tcii18n.template import flask_methods
+
+import gfm
+
+from . import app
+
+
+def get_translations_file():
+    supported_languages = app.config.get('TRANSLATION_FILES').keys()
+    lang = (request.accept_languages.best_match(supported_languages) or
+            supported_languages[0])
```

Comment:
Are we sure that we'll always have at least 1 language in `app.config.get('TRANSLATION_FILES').keys()` ?